building/trusted_applications.rst

```diff
@@ -347,6 +347,14 @@ sequence:
         }
     }
 
+.. warning::
+  As stated in the Global API documentation (v3.1.12, section 4.11.1), memory 
```

Comment:
That's v1.3.1 not v3.1.12. Not sure if there is a newer version though, but it doesn't matter much.